index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile-Main</title>
```

Comment:
- **[OPTIONAL]** Kindly consider using a title that represents what your website is all about 🆗. This is what your users will see in their browser tabs and it'll give them an idea of what your site stands for 💪🏽.